pkg/proof/proof_test.go

```diff
@@ -123,8 +123,8 @@ func TestNewShareInclusionProof(t *testing.T) {
 
 	type test struct {
 		name          string
-		startingShare int
-		endingShare   int
+		startingShare int64
```

Comment:
why are we changing these to `int64` then converting them back to `int` in line 223?

Reply:
because `ParseNamespace` now accepts an `int64`